.github/workflows/ci.yml

```diff
@@ -40,12 +40,31 @@ jobs:
         # Wait for elasticsearch to report healthy before continuing.
         # see https://github.com/actions/example-services/blob/master/.github/workflows/postgres-service.yml#L28
         options: -e "discovery.type=single-node" --expose 9200 --health-cmd "curl localhost:9200/_cluster/health" --health-interval 10s --health-timeout 5s --health-retries 10
+      localstack:
+        image: localstack/localstack:0.10.8
+        env:
+          SERVICES: kinesis:4568,dynamodb:4569
```

Comment:
I'd love for this to be DRYer as it's repeating the `docker-compose.yml` but not sure how.

Reply:
I think that while both of these are defined with YAML this will repetition will be necessary, since you can't share any configuration across documents.

Reply:
I think there might be a way with [`options`](https://help.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobsjob_idservicesoptions) as they follow the [docker options spec](https://docs.docker.com/engine/reference/commandline/create/#options). Something like `--env-file`? Alas, I [couldn't get it to work](https://github.com/guardian/grid/runs/573750783?check_suite_focus=true#step:2:65).